src/AppBundle/Controller/AdmissionAdminController.php

```diff
@@ -370,4 +370,31 @@ public function showApplicationAction(Application $application)
             'application' => $application,
         ));
     }
+
+    public function showTeamInterestAction(Request $request, $departmentId = null)
```

Comment:
Her er det mulig å hente `Department` direkte, altså `public function showTeamInterestAction(Request $request, Department $department = null)`. Kan dette gjøre logikken for å finne riktig department enklere?